app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import random
+
+
+def flip_coin():
+    outcomes = dict()
+    for i in range(11):
+        outcomes[i] = 0
+
+    for _ in range(10000):
+        heads = 0
+        for i in range(10):
+            if random.randint(0, 10) & 1:
```

Comment:
You have a mistake here, build a graph to visualize it. Also, don't use the bitwise operator here, it is very implicit and unclear.